bitnami/dataplatform-bp2/values.yaml

```
@@ -481,7 +481,15 @@ logstash:
                 values:
                   - '{{ .Release.Name }}'
           topologyKey: "kubernetes.io/hostname"
-
+  ## @param extraEnvVars Array containing extra env vars to configure Logstash
```

This should fix the CI
```suggestion
  ## @param logstash.extraEnvVars Array containing extra env vars to configure Logstash
```

---

Hi @miguelaeh, Thanks for the suggestion. Resolved this.